bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/igrins2/SeqConfigIgrins2.java

```diff
@@ -0,0 +1,40 @@
+package edu.gemini.spModel.gemini.igrins2;
+
+import edu.gemini.pot.sp.ISPNodeInitializer;
+import edu.gemini.pot.sp.ISPSeqComponent;
+import edu.gemini.pot.sp.SPComponentType;
+import edu.gemini.spModel.data.property.PropertyFilter;
+import edu.gemini.spModel.data.property.PropertyProvider;
+import edu.gemini.spModel.data.property.PropertySupport;
+import edu.gemini.spModel.gemini.init.ComponentNodeInitializer;
+import edu.gemini.spModel.seqcomp.SeqConfigNames;
+import edu.gemini.spModel.seqcomp.SeqConfigObsBase;
+
+import java.beans.PropertyDescriptor;
+import java.util.Collections;
+import java.util.Map;
+
+final public class SeqConfigIgrins2 extends SeqConfigObsBase implements PropertyProvider {
```

Comment:
I'm not totally sure this is needed, it was essentially copied from the Ghost model

Reply:
If it has an iterator, even a simple one where you can only change the exposure time, then I think we need one of these.